.gitignore

```diff
@@ -1,3 +1,10 @@
 build/
 *.user
 src/appMain/sdl_preloaded_pt.json
+src/3rd_party-static/openssl-1.0.2/
+src/3rd_party-static/libusbx-1.0.17/
+*.pyc
+src/components/utils/src/threads/thread.h
+src/components/utils/src/threads/thread_delegate.h
+src/components/media_manager/src/audio/from_mic_to_file_recorder_thread.h
+src/components/utils/src/socket.h
```

Comment:
@anosach-luxoft, 
After discussion with @nk0leg we have recognized the root issue
 
In case of using `QObject` inheritance we need to handle source and header file in the same folder.
But we pcan try to avoid such workaround with cmake and qt](https://stackoverflow.com/questions/19866424/cmake-and-qt5-automoc-error).

Splitted to [APPLINK-25004](https://adc.luxoft.com/jira/browse/APPLINK-25004)